iot_api/user_api/resources/ResourceUsage.py

```diff
@@ -136,6 +136,7 @@ def get(self):
         assets = [{
             'id': dev.id,
             'hex_id': dev.hex_id,
+            'dev_addr': dev.dev_addr,
```

Comment:
Same above, but this time the value has to be added to the get single resource endpoint (_ResourceUsageInformationAPI_)